returnn/hdf.py

```diff
@@ -281,3 +283,68 @@ def run(self):
             audio.close()
 
         out_hdf.close()
+
+
+class RasrAlignmentDumpHDFJob(Job):
+    """
+    This Job reads Rasr alignment caches and dump them in hdf files.
+    """
+
+    def __init__(
+        self,
+        alignment_caches: List[tk.Path],
+        allophone_file: tk.Path,
+        state_tying_file: tk.Path,
+        data_type: type = np.uint16,
+        returnn_root: Optional[tk.Path] = None,
+    ):
+        """
+        :param alignment_caches: e.g. output of an AlignmentJob
+        :param allophone_file: e.g. output of a StoreAllophonesJob
+        :param state_tying_file: e.g. output of a DumpStateTyingJob
```

Comment:
```suggestion
        :param state_tying_file: e.g. output of a DumpStateTyingJob
        :param data_type: type that is used to store the data
```
The added `:param data_type:` from my last comment was missing in the last commit, so here again :P